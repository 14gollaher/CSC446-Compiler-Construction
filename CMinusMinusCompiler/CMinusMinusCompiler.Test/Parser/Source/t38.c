int main() {
	return 5 * 3;
}

hello
