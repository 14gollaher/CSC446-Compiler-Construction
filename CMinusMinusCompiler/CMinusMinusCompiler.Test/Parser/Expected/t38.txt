ERROR: Line 6 Unexpected tokens in source file, expected End-of-File Token
Completed processing t38.c